Monika After Story/game/script-topics.rpy

```diff
@@ -4897,9 +4897,9 @@ label monika_panties:
                m 1rkc "I know that some topics are best kept to self until the right time comes."
                m 1rksdld "Not to mention it's controversial..."
                m 1ekbfa "..."
-               m 1eka "In any case, {i}don't be afraid to tell me about your pleasure, okay [player]?"
+               m 1eka "In any case,{i}don't be afraid to tell me about your pleasure, okay [player]?"
```

Comment:
same here with {/i}.  also add that space back or there won't be a space between the , and "don't"